website/source/docs/apps/index.html.md

```diff
@@ -5,7 +5,7 @@ sidebar_current: "docs-apps"
 description: |-
   The application type in the Appfile
   is the most important piece of information that tells Otto how to function.
-  Otto uses this information to know how to develop, build, and deploy
```

Comment:
This is an oxford comma and is preferred over the non-oxford version, especially for non-native speakers. I am against removing it, and it's grammatically correct.